cla_public/templates/base.html

```diff
@@ -54,15 +54,9 @@
     {% endif %}
   {% endif %}
   <div class="laa-flee-button-container
-    {% if ('n43n3' in session.checker.diagnosis_previous_choices) or
-        ('n97' in session.checker.diagnosis_previous_choices) or
-        ('n86' in session.checker.diagnosis_previous_choices) or
-        ('n149' in session.checker.diagnosis_previous_choices) or
-        ('n18' in session.checker.diagnosis_previous_choices) or
-        ('n19' in session.checker.diagnosis_previous_choices) or
-        ('n88' in session.checker.diagnosis_previous_choices)
-    %}
+    {% if is_quick_exit_enabled(node, session) %}
       laa-fee-button-enabled
+      {{ is_quick_exit_enabled(node, session) }}
```

Comment:
This value has been added to the output in the refactor (ie you've replaced the `if` condition but also added the matching category). Is that intentional? Was it a debugging thing? Or does it achieve some undocumented aim?

Reply:
A debugging thing...  one of the things I forgot about over Easter...